spec/services/virtual_hearings/sequence_conference_id_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+describe VirtualHearings::SequenceConferenceId do
+  context ".next" do
+    it "returns values in sequence" do
+      first_value = described_class.next
+      second_value = described_class.next
+      expect(first_value.to_i).to eq(second_value.to_i - 1)
+    end
+
+    it "returns a seven character string starting with '0'" do
+      check_value = described_class.next
+      expect(check_value.class).to eq String
+      expect(check_value.length).to eq 7
+      expect(check_value[0]).to eq "0"
```

Comment:
You could _optionally_ modify this to 
```suggestion
      (0..5).each { |index| expect(check_value[index]).to eq "0" }
```

Reply:
I think I'm find just testing for the first zero...just want to make sure the zero-padding is working.